core/src/main/java/oracle/weblogic/deploy/util/WLSBeanHelp.java

```diff
@@ -0,0 +1,565 @@
+/*
+ * Copyright (c) 2022, Oracle Corporation and/or its affiliates.  
+ * Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
+package oracle.weblogic.deploy.util;
+
+import java.beans.BeanDescriptor;
+import java.beans.BeanInfo;
+import java.beans.FeatureDescriptor;
+import java.beans.PropertyDescriptor;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.HashMap;
+
+/**
+ * Helper for getting attribute descriptions from a WL mbean.
+ *
+ * Includes an undocumented main intended for ad-hoc printing of a particular bean or bean property help.
+ */
+
+public class WLSBeanHelp {
```

Comment:
Just a general comment but I think we try to keep the class organized so that methods of the same visibility and scope are grouped together.  For example:

- public static 
- static
- private static

All utility classes should generally have a private constructor to prevent developers from accidentally trying to instantiate it.  I suspect Sonar will flag some of these issues...